com.unity.render-pipelines.high-definition/Documentation~/Light-Component.md

```diff
@@ -150,15 +150,15 @@ These settings define the emissive behavior of your Light. You can set the Light
 | **Range Attenuation**     | Enable the checkbox to make this Light shine uniformly across its range. This stops light from fading around the edges. This setting is useful when the range limit is not visible on screen, and you do not want the edges of your light to fade out. This property is available for all **Light Types** except **Directional**.<br />This property only appears when you enable [additional properties](More-Options.md) for this section. It is only available in Realtime or Mixed light **Mode** for **Type** Area. |
 | **Fade Distance**         | The distance between the Light source and the Camera at which the Light begins to fade out. Measured in meters. This property is available for all **Light Types** except **Directional**.<br />This property only appears when you enable [additional properties](More-Options.md) for this section. It is only available in Realtime or Mixed light **Mode**. |
 | **Intensity Multiplier**                | A multiplier that gets applied to the intensity of the Light. Does not affect the intensity value, but only gets applied during the evaluation of the lighting. You can also modify this property via [Timeline](https://docs.unity3d.com/Manual/TimelineSection.html), Scripting or [animation](https://docs.unity3d.com/Manual/animeditor-AnimatingAGameObject.html). The parameter lets you fade the Light in and out without having to store its original intensity.<br />This property only appears when you enable [additional properties](More-Options.md) for this section. It is only available in Realtime or Mixed light **Mode**. |
 | **Display Emissive Mesh** | Enable the checkbox to make Unity automatically generate a Mesh with an emissive Material using the size, colour, and intensity of this Light. Unity automatically adds the Mesh and Material to the GameObject the Light component is attached to. This property is available for **Rectangle** and **Tube** Lights.<br />This property only appears when you enable [additional properties](More-Options.md) for this section. (In case of an IES profile and a cookie used at the same time, only the cookie will be displayed). |
-| **Include For Ray Tracing ** | Enable the checkbox to make this Light active when the camera has the **Ray Tracing** [Frame Setting](Frame-Settings.md) enabled. This applies to both rasterization and [ray tracing](Ray-Tracing-Getting-Started.md) passes.<br />This property only appears when you enable [additional properties](More-Options.md) for this section. It is only available in Realtime or Mixed light **Mode**. |
+| **Include For Ray Tracing** | Enable the checkbox to make this Light active when the camera has the **Ray Tracing** [Frame Setting](Frame-Settings.md) enabled. This applies to both rasterization and [ray tracing](Ray-Tracing-Getting-Started.md) passes.<br />This property only appears when you enable [additional properties](More-Options.md) for this section. It is only available in Realtime or Mixed light **Mode**. |
```

Comment:
Removed passive voice: 

| **Include For Ray Tracing** | Enable the checkbox to make this Light active when you enable the **Ray Tracing** [Frame Setting](Frame-Settings.md) on the Camera. This applies to rasterization and [ray tracing](Ray-Tracing-Getting-Started.md) passes.<br />This property only appears when you enable [additional properties](More-Options.md) for this section. It is only available in Realtime or Mixed light **Mode**. |